src/Sulu/Component/Content/Document/Behavior/SyncronizeBehavior.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Content\Document\Behavior;
+
+use Sulu\Component\DocumentManager\Behavior\Path\BasePathBehavior;
+
+/**
+ * TODO: Document me.
+ */
+interface SyncronizeBehavior
+{
+    const SYNCED_FIELD = 'synced';
```

Comment:
Should probably move this constant to the subscriber as with all other constants of this nature.